.travis.yml

```diff
@@ -7,47 +7,13 @@ os: osx
 osx_image: xcode8.3
 cache:
   - cocoapods
-
+florentvilmart@me.com
```

Comment:
This is causing a syntax for your travis.yml. There are a couple other things I noticed in lint that I still need to look at here.